tests/ui/lint/non-local-defs/cargo-update.rs

```diff
@@ -0,0 +1,13 @@
+//@ check-pass
+//@ edition:2021
+//@ aux-build:non_local_macro.rs
+//@ rustc-env:CARGO_CRATE_NAME=non_local_def
+
+extern crate non_local_macro;
+
+struct CargoUpdate;
```

Comment:
My only request is maybe changing the test and struct name to be more about a local type in a non-local macro

Reply:
I changed the struct name but not test name since this test is specifically here to check that we correctly emit the `cargo update` suggestion.

Reply:
Oh, lol, I totally misinterpreted the name then. Carry on.